### DISCUSSION TITLE
Problem with DeDRM_tools_10.0.3 and NOOK drmed files

### DISCUSSION THREAD

**Discussion by DrakBibliophile** [noDRM/DeDRM_tools#342] [Q&A] [answered]
OK, I recently purchased a DRMed Nook eBook.

I had also downloaded the DeDRM_tools_10.0.3 and the DeDRM_plugin to my calibre 5.44 app.

While Calibre appeared to DeDRM the Nook eBook, when I opened in Calibre I'm getting garbage and Adobe Digital Editions 4.5.11.187303 couldn't even open it.

Am I to add B&N info into Calibre?

If so, where in Calibre do I add it and what info do I add?

Thank you.

**Comment by ElleKayEm** [comment]
Look in DeDRM's customization.  Easiest way is if you have the Windows Nook app on the same machine and user, DeDRM can grab your BN key from that.

**Reply by DrakBibliophile** [reply]
That's the problem.  I do have the Windows Nook app and copied the Nook ebook to my "Add to Calibre" folder.

I saw nothing in the "Read Me" text doc about Customization for Nook.

**Comment by ElleKayEm** [comment]
In calibre go to Preferences > Plugins.  Select the DeDRM plugin under File Type plugins.  Then click customize plugin > ADE PassHash (B&N) ebooks.  Click green plus sign and select Extract key from Nook Windows appplication.  See the help in that dialog box for more details.

**Comment by DrakBibliophile** [comment, marked as answer]
Thanks!  That Worked!
